### PR TITLE
Avoid Flash of Invisible Text

### PR DESCRIPTION
**Pull request by robinandeer**
I ran Lighthouse and it reported an error that we don't avoid FOIT. The resolution for most browsers is to add:

```
font-display: swap
```

Reference: https://web.dev/font-display/?utm_source=lighthouse&utm_medium=devtools#how-to-avoid-showing-invisible-text

I also noticed that we don't really need to include multiple font formats. So I removed the old "woff" font format. "woff2" is supported by 95%+ users (https://caniuse.com/woff2)

### CODE REVIEW

**Comment by simonauner** [comment]
Interesting that browsers solved "Flash of unstyled text" by waiting to render until the font is loaded, but now web devs _want_ it to render unstyled instead 😂 

(only an observation, I agree that it's better to show unstyled text than no text at all)

File: src/fonts/index.ts [hunk 1]
```diff
@@ -27,10 +27,8 @@ const getCdnFontFaces = () =>
     font-family: "${fontName}";
     font-style: ${flavour.style};
     font-weight: ${flavour.weight};
-    src:
-      url("https://cdn.hedvig.com/identity/fonts/${fontName}.woff2") format("woff2"),
-      url("https://cdn.hedvig.com/identity/fonts/${fontName}.woff") format("woff")
-    ;
+    src: url("https://cdn.hedvig.com/identity/fonts/${fontName}.woff2") format("woff2");
```
Comment (gustaveen) [inline src/fonts/index.ts:30]: Thought the difference in support was bigger but it's only like 1% so I think it's fine :) But wondering if the browser actually downloads the 2 formats? Looks like it resolves (chrome at least) what format to pick right?
Screenshot 2022-05-09 at 10 16 15

**Review by gustaveen — APPROVED** [review]
Great Robin! 👌 It's a UX improvement for sure and I suppose most up to date browsers follow this patterns as well. Might be a good thing to look into the fallback fonts use as well, if we want to have something more specific than `serif` and  `sans-seif`

File: src/fonts/index.ts (hunk 1 above)
Comment (robinandeer) [reply src/fonts/index.ts:30]: I'm pretty sure the browser is smart enough to only download once. But I just thought we would simplify and stick to committed browser support: https://www.notion.so/hedviginsurance/Browser-support-for-hedvig-com-67469ca06ff94433be3e9a3840a80744

**Pull request merged**